core/domain/exp_services.py

```diff
@@ -1912,3 +1917,54 @@ def discard_draft(exp_id, user_id):
         exp_user_data.draft_change_list_last_updated = None
         exp_user_data.draft_change_list_exp_version = None
         exp_user_data.put()
+
+
+def get_state_id_mapping_model(exp_id, exp_version, strict=False):
```

Comment:
From reading this function, it's actually getting a domain object, right? So it should be: get_state_id_mapping(...)

Also, I don't think we should have the `strict` arg here.

Reply:
Removed strict.